<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryclientes</class>
 <widget class="QWidget" name="queryclientes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nome:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_name"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>NIF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_nif"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>ID_MOTO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="check_ative">
       <property name="text">
        <string>Ativo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_consultar">
       <property name="text">
        <string>Consultar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
